### ISSUE TITLE
handle None Type in messages

### ISSUE THREAD

**Issue by tklebel** [tklebel/humidity_tracking#1]
Trace:

```python
Traceback (most recent call last):
  File "/home/pi/.local/lib/python3.7/site-packages/telegram/ext/jobqueue.py", line 270, in tick
    job.run(self._dispatcher)
  File "/home/pi/.local/lib/python3.7/site-packages/telegram/ext/jobqueue.py", line 399, in run
    self.callback(CallbackContext.from_job(self, dispatcher))
  File "bot.py", line 42, in monitor_humidity
    humidity_pretty = "{0:0.1f}%".format(humidity)
TypeError: unsupported format string passed to NoneType.__format__
```

**Comment by tklebel** [comment]
This is still not fixed:

> Reading data and writing to database every 30 seconds.
Terminating...
Database connections closed.
Display cleared.
Traceback (most recent call last):
  File "read_sensor.py", line 81, in <module>
    main()
  File "read_sensor.py", line 73, in main
    lcd.message('Temp={0:0.1f}*C\n'.format(temperature))
TypeError: unsupported format string passed to NoneType.__format__
2020-01-04 12:58:13,897 - JobQueue - ERROR - An uncaught error was raised while executing job monitor_humidity
Traceback (most recent call last):
  File "/home/pi/.local/lib/python3.7/site-packages/telegram/ext/jobqueue.py", line 270, in tick
    job.run(self._dispatcher)
  File "/home/pi/.local/lib/python3.7/site-packages/telegram/ext/jobqueue.py", line 399, in run
    self.callback(CallbackContext.from_job(self, dispatcher))
  File "bot.py", line 45, in monitor_humidity
    humidity = results[1]
TypeError: 'NoneType' object is not subscriptable

The recursive loop needs to be tested again.

**Comment by tklebel** [comment]
> 2020-01-04 21:54:08,094 - JobQueue - ERROR - An uncaught error was raised while executing job monitor_humidity
Traceback (most recent call last):
  File "/home/pi/.local/lib/python3.7/site-packages/telegram/ext/jobqueue.py", line 270, in tick
    job.run(self._dispatcher)
  File "/home/pi/.local/lib/python3.7/site-packages/telegram/ext/jobqueue.py", line 399, in run
    self.callback(CallbackContext.from_job(self, dispatcher))
  File "bot.py", line 36, in monitor_humidity
    humidity = results[1]
TypeError: 'NoneType' object is not subscriptable
Reading data and writing to database every 30 seconds.
We got no reading, but ``humidity = None & temp = None`` , trying again.

> Terminating...
Database connections closed.
Display cleared.
Traceback (most recent call last):
  File "read_sensor.py", line 87, in <module>
    main()
  File "read_sensor.py", line 68, in main
    time, humidity, temperature = get_data()
TypeError: cannot unpack non-iterable NoneType object

**Comment by tklebel** [comment]
this should be fixed with 7bf86c4
